### DISCUSSION TITLE
What is fallbackExclusionList?

### DISCUSSION THREAD

**Discussion by coo-kiee** [yarnpkg/berry#5386] [Q&A]
When i added peerDependencies my monorepo package(@common/hooks) like this:
![image](https://user-images.githubusercontent.com/76735567/231670588-39415939-3059-430e-8e07-a68965454017.png)

After excute yarn(I used yarn berry)

my .pnp.cjs changed to like this:
// before
![image](https://user-images.githubusercontent.com/76735567/231670452-f6be6adb-9108-4608-8441-e3f1da1adfa2.png)
["@common/hooks", ["workspace:packages/common-hooks"]],\

// after
![image](https://user-images.githubusercontent.com/76735567/231670422-9069e32a-c81a-430a-94cd-a53d4736a22f.png)
["@common/hooks", ["virtual:325a776a1f46a561d00b86ada7f47d0925448b6325933dde0a3fcb4c69d3aab43e415e273c5eaa8bc2fde1d7f0b4a58f8f0b6d7e73ad61de0369a6613827f4c7#workspace:packages/common-hooks", "workspace:packages/common-hooks"]],\

And Finally, my app didn't intellisense packages such as auto import

Anyone tell me why it is?

**Comment by merceyz** [comment]
> What is fallbackExclusionList?

https://yarnpkg.com/advanced/pnp-spec#fallbackExclusionList

> And Finally, my app didn't intellisense packages such as auto import
Anyone tell me why it is?

Not without more details.
